validator/project_validator.go

```diff
@@ -585,6 +590,28 @@ func ensureReferentialIntegrity(project *model.Project, distroIDs []string, dist
 				}
 			}
 			buildVariantTasks[task.Name] = true
+
+			if task.IsGroup {
+				for _, tgTask := range tgNameTasksMap[task.Name] {
+					if _, ok := buildVariantTasks[tgTask]; ok {
+						errs = append(errs,
+							ValidationError{
+								Message: fmt.Sprintf("task '%s' in build variant '%s' is already referenced in '%s' task group",
```

Comment:
Level: Error is default when we don't specify but I think it would be more readable if we explicitly set it.